extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/HudiCatalogRangerSparkExtensionSuite.scala

```diff
@@ -127,6 +127,7 @@ class HudiCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
       interceptContains[AccessControlException](
         doAs(someone, sql(s"ALTER TABLE $namespace1.$table1 ADD COLUMNS(age int)")))(
         s"does not have [alter] privilege on [$namespace1/$table1]")
+      sql("set hoodie.schema.on.read.enable=false")
```

Comment:
@yaooqinn After check it's caused by I forgot to set this parameter back.

Reply:
Let's make sure to reach this line after L126 with a try-finally block?

Reply:
Done